test/Validation.Tests/RequiresTests.cs

```diff
@@ -5,7 +5,9 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Threading.Tasks;
+
```

Comment:
I'm not a fan of this separation because my IDE will not maintain it when the language service adds these automatically, or executes "remote and sort usings". Is this an extension you have, or an IDE setting? If there were an .editorconfig setting for it that would lead the IDE to maintain this, I would consider it (but I don't see much value in it anyway).

Reply:
Pretty sure this is from a plugin. I'll see if I can figure out which one and disable it (or see if it supports an .editorconfig setting), if not, I'll manually remove the blank lines outside of VS.

Reply:
I was wrong. It seems to be a default option in Visual Studio (Options|Text Editor|C#|Advanced => Using Directives/Separate using directive groups). Now to figure out what the .editorconfig setting is for it.